src/components/Slider/index.tsx

```diff
@@ -11,7 +11,7 @@ interface SliderProps {
   items?: JSX.Element[];
   isLoading: boolean;
   isEmpty?: boolean;
-  emptyMessage?: string;
+  emptyMessage?: string | React.ReactNode;
```

Comment:
ReactNode also takes a string, so can just make this accept only React.ReactNode
```suggestion
  emptyMessage?: React.ReactNode;
```